tests/monotouch-test/CoreGraphics/CGImagePropertiesExifTest.cs

```diff
@@ -0,0 +1,131 @@
+using System;
+using System.IO;
+
+using CoreGraphics;
+using CoreImage;
+using Foundation;
+#if MONOMAC
+using AppKit;
+#else
+using UIKit;
+#endif
+
+using NUnit.Framework;
+
+namespace monotouchtest.CoreGraphics {
+	[TestFixture]
+	[Preserve (AllMembers = true)]
+	public class CGImagePropertiesExifTest {
+		[Test]
+		public void ConstructorAndBasicPropertiesTest ()
+		{
+			// Test default constructor
+			var exif = new CGImagePropertiesExif ();
+			Assert.That (exif, Is.Not.Null, "Default constructor should create a valid instance");
+
+			// Test setting and getting basic properties
+			exif.Aperture = 2.8f;
+			Assert.That (exif.Aperture, Is.EqualTo (2.8f).Within (0.001f), "Aperture property should be settable and gettable");
+
+			exif.ExposureTime = 0.125f;
+			Assert.That (exif.ExposureTime, Is.EqualTo (0.125f).Within (0.001f), "ExposureTime property should be settable and gettable");
+
+			exif.Flash = true;
+			Assert.That (exif.Flash, Is.EqualTo (true), "Flash property should be settable and gettable");
+
+			exif.PixelXDimension = 1920;
+			Assert.That (exif.PixelXDimension, Is.EqualTo (1920), "PixelXDimension property should be settable and gettable");
+
+			exif.PixelYDimension = 1080;
+			Assert.That (exif.PixelYDimension, Is.EqualTo (1080), "PixelYDimension property should be settable and gettable");
+		}
+
+		[Test]
+		public void ConstructorWithDictionaryTest ()
+		{
+			using (var dict = new NSMutableDictionary ()) {
```

Comment:
Use the "using" declaration syntax instead of the using block syntax to reduce nesting.